app/Http/Controllers/AuthenticateController.php

```diff
@@ -32,6 +32,17 @@ public function authenticate(Request $request){
                    ]);
   }  
 
+  //refresh route for the front end
+  public function user(){
+    $user = JWTAuth::parseToken()->toUser();
+    $token = JWTAuth::parseToken()->refresh(); 
+    $data = array();
+    $data["user"] = $user;
+    return response($data)->withHeaders([
+                      'Authorization'=> 'Bearer: '.$token
```

Comment:
I hear ya and I feel ya, but technically this is a self issued authorization token and such we follow http://self-issued.info/docs/draft-ietf-oauth-v2-bearer.html#authz-header